### PR TITLE
OssStorage provides __init__ arguments

### PR DESCRIPTION
**Pull request by TonyPythoneer**
Hello, @baiyubin @abo321 

I hope OssStorage can populate arguments when initializing instance. It's more flexible for developer using. Please review this. Thanks!

### CODE REVIEW

**Comment by TonyPythoneer** [comment]
請問負責這個 repo 的聯絡窗口是誰？能不能出來回應一下？

cc @baiyubin @abo321

**Comment by TonyPythoneer** [comment]
已經過了一個禮拜了，請問為什麼沒有人回應？

cc @baiyubin @abo321 @oldratlee

**Comment by abo321** [comment]
@TonyPythoneer ,

Really appreciate your feedback on this. Could you please describe your use case in detail for developer? Initially by design we would like to config AK, bucket and endpoint firstly in env variables or in setting.py.

Again, thank you!

**Comment by TonyPythoneer** [comment]
Hi @abo321,

Because I expect each bucket can store specified file type, I need to initialize storage by different oss key and mange that, for example:
1. Pictures store in `picture bucket`
2. Audios store in ` audio bucket`
3. Vedios store in ` vedio bucket`

If keep original, the files will store in the same bucket. It's not something I wanted.

I requires offering more flexible initializing methods to match my demand.

Thansks for your response.

**Comment by abo321** [comment]
It sounds reasonable. Please provide test code as you write function code. And see my comments inline.

File: django_oss_storage/backends.py [hunk 1]
```diff
@@ -26,6 +26,17 @@
 from .defaults import logger
 
 
+def _get_config(name):
```
Comment (abo321) [inline django_oss_storage/backends.py]: Please provide test code for this new function.

File: django_oss_storage/backends.py [hunk 2]
```diff
@@ -45,11 +56,11 @@ class OssStorage(Storage):
     Aliyun OSS Storage
     """
 
-    def __init__(self):
-        self.access_key_id = self._get_config('OSS_ACCESS_KEY_ID')
-        self.access_key_secret = self._get_config('OSS_ACCESS_KEY_SECRET')
-        self.end_point = _normalize_endpoint(self._get_config('OSS_ENDPOINT'))
-        self.bucket_name = self._get_config('OSS_BUCKET_NAME')
+    def __init__(self, access_key_id=None, access_key_secret=None, end_point=None, bucket_name=None):
```
Comment (abo321) [inline django_oss_storage/backends.py]: Also need test for these default parameters.

**Comment by TonyPythoneer** [comment]
Hello @abo321 , I had added testcases, please review it.

By the way, when do you add @CLAassistant ?

**Comment by TonyPythoneer** [comment]
@abo321 What do you mean?

**Comment by abo321** [comment]
I just reviewed your code and think it can be merged to the main stream. Then the owner will merge that. You don't need to know that since this is just the process here.

**Comment by TonyPythoneer** [comment]
@abo321 OK, I hope it can merge into master branch as soon as possible.

**Pull request merged**